guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -1,6 +1,26 @@
 [id="performing-post-upgrade-tasks_{context}"]
 = Performing post-upgrade tasks
 
-Optional: If the default provisioning templates have been changed during the upgrade, recreate any templates cloned from the default templates.
+* Optional: If the default provisioning templates have been changed during the upgrade, recreate any templates cloned from the default templates.
 If the custom code is executed before and/or after the provisioning process, use custom provisioning snippets to avoid recreating cloned templates.
 For more information about configuring custom provisioning snippets, see {ProvisioningDocURL}Creating_Custom_Provisioning_Snippets_provisioning[Creating Custom Provisioning Snippets] in _{ProvisioningDocTitle}_.
+ifdef::katello,orcharhino,satellite[]
+* Pulp is introducing more data about container manifests to the API.
+This information allows Katello to display manifest labels, annotations, and information about the manifest type, such as if it is bootable or represents flatpak content.
+As a result, migrations must be performed to pull this content from manifests into the database.
+endif::[]
+ifdef::katello,orcharhino[]
++
+This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following command after upgrading for a pre-migration.
```

Comment:
```suggestion
This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following command for a pre-migration after upgrading:
```
This order of the phrases looks cleaner to me. Plus a colon, because the sentence introduces a block code.